src/io/fabric8/Fabric8Commands.groovy

```diff
@@ -955,37 +944,36 @@ def openShiftImageStreamExists(String name){
             def result = sh(returnStdout: true, script: 'oc describe is ${name} --namespace openshift')
             if (result && result.contains(name)){
                 echo "ImageStream  ${name} is already installed globally"
-                return true;
+                return true
             }else {
                 //see if its already in our namespace
-                def namespace = kubernetes.getNamespace();
+                def namespace = kubernetes.getNamespace()
                 result = sh(returnStdout: true, script: 'oc describe is ${name} --namespace ${namespace}')
                 if (result && result.contains(name)){
                     echo "ImageStream  ${name} is already installed in project ${namespace}"
-                    return true;
+                    return true
                 }
             }
         }catch (e){
             echo "Warning: ${e} "
         }
     }
-    return false;
+    return false
 }
 
 @NonCPS
 def openShiftImageStreamInstall(String name, String location){
     if (openShiftImageStreamExists(name)) {
         echo "ImageStream ${name} does not exist - installing ..."
         try {
-            def result = sh(returnStdout: true, script: 'oc create -f  ${location}')
```

Comment:
why is this deleted?

Reply:
Oops. The linter warned that the variable is unused and removed it. It should be around. I caught a bunch of these when I checked the diff in the end, but this one slipped through.